python/ml_croissant/scripts/load.py

```diff
@@ -57,7 +56,27 @@ def main(argv):
     num_records = FLAGS.num_records
     debug = FLAGS.debug
     update_output = FLAGS.update_output
+    return load(
```

Comment:
I wonder if this change would have been better placed in a separated PR, given that is not related to Image processing?